app/instance-initializers/axe-component.js

```diff
@@ -40,12 +40,13 @@ export function initialize(application) {
     turnAuditOff: false,
 
     /**
-     * An array of classNames to add to the component when a violation occurs.
+     * An array of classNames (or a space-separated string) to add to the component when a violation occurs.
```

Comment:
Not unreasonable. If we're expecting people to use this as part of public API we should probably add an example to the docs.